doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,984 @@
+    * Removed all read callbacks from sensor code because of an issue of read callbacks not working properly when used with LwM2M observations.
+      This is due to the fact that the engine does not know when data is changed.
+    * Sensor samples are now enabled by default for Thingy:91 and disabled by default on nRF9160 DK.
+
+* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
+
+  * Updated:
+
+   * Sample moved from :file:`samples/nrf9160/nrf_cloud_rest_cell_pos` to :file:`samples/nrf9160/nrf_cloud_rest_cell_location`.
+
+* :ref:`modem_shell_application` sample:
+
+  * Added:
+
+    * The functionality where **LED 1** (nRF9160 DK)/Purple LED (Thingy:91) is lit for five seconds indicating that a current location has been acquired by using the ``location get`` command.
+    * Overlay files for nRF9160 DK with nRF7002 EK to enable Wi-Fi scanning support.
+      With this configuration, you can, for example, obtain the current location using the ``location get`` command.
+    * Support for new GCI (Global Cell ID) search types for ``link ncellmeas`` command, which are supported by the modem firmware versions from and including 1.3.4.
+    * Handling for the new data receive events in the :ref:`lib_nrf_cloud` library.
+    * Support for connecting to cloud using the :ref:`lib_lwm2m_client_utils` library.
+      An overlay file is provided for building with the LwM2M support and an optional overlay to enable P-GPS.
+
+  * Removed A-GPS and P-GPS processing; it is now handled by the :ref:`lib_nrf_cloud` library.
+
+* :ref:`location_sample` sample:
+
+  * Added overlay files for nRF9160 DK with nRF7002 EK to obtain the current location by using Wi-Fi scanning results.
+
+* :ref:`lte_sensor_gateway` sample:
+
+  * Added handling for the new data receive events in the :ref:`lib_nrf_cloud` library.
+  * Removed A-GPS and P-GPS processing; it is now handled by the :ref:`lib_nrf_cloud` library.
+
+* :ref:`nrf_cloud_mqtt_multi_service` sample:
+
+  * Added:
+
+    * Support for the :ref:`lib_location` library Wi-Fi location method with the nRF7002 EK.
+    * Handling for the new data receive events in the :ref:`lib_nrf_cloud` library.
+    * Board overlay file for nRF9160 DK with external flash.
+    * Overlay file to enable P-GPS data storage in external flash.
+
+  * Removed A-GPS and P-GPS processing; it is now handled by the :ref:`lib_nrf_cloud` library.
+
+Trusted Firmware-M (TF-M) samples
+---------------------------------
+
+* Added:
+
+  * :ref:`tfm_psa_test` for validating compliance with Arm PSA Certified requirements.
+  * :ref:`tfm_regression_test` to run secure and non-secure Trusted Firmware-M (TF-M) regression tests.
+  * :ref:`tfm_psa_template`, providing a template for Arm PSA best practices on nRF devices.
+  * :ref:`provisioning_image` sample that provisions the device in a manner compatible with Trusted Firmware-M (TF-M).
```

Comment:
We can maybe add for information regarding the two samples here:
tfm_psa_template: providing a template for Arm PSA best practices on nRF devices and enforcing correct transition/usage of the PSA lifecycle states.  


provisioning_image: sample that provisions the PSA platform root of trust parameters (such as the PSA Implementation ID and  lifecycle state)  in a manner compatible with Trusted Firmware-M (TF-M).

Reply:
Added this, @Vge0rge